messages/id.json

```diff
@@ -0,0 +1,317 @@
+{
+  "titles" : {
+    "privacyPolicy": "Kebijakan Privasi",
+    "terms": "Syarat dan Ketentuan",
+    "pdfSnake": "PDF Snake",
+    "pdfSnakeVersion": "PDF Snake {version}"
+  },
+  "topNav" : {
+    "home": "Beranda",
+    "about": "Tentang",
+    "signIn": "Masuk",
+    "myAccount": "Akun Saya"
+  },
+  "dropTarget" : {
+    "dropHere": "Tarik sebuah file PDF ke sini atau",
+    "browse": "Pilih File untuk Memulai",
+    "docsStayPrivate": "Dokumen-dokumen Anda tetap terjaga kerahasiaannya karena mereka tidak pernah meninggalkan perangkat Anda.",
+    "help": "Bantuan"
+  },
+  "about" : {
+    "privacy": "Baca {link} kami.",
+    "terms": "Dengan menggunakan situs web ini, Anda setuju dengan {link} kami.",
+    "plugIn": "PDF Snake 6 adalah versi daring baru dari <i>Plug-In</i> Adobe Acrobat yang terkenal, {link}.",
+    "questions": "Kami senang menerima pertanyaan dan komentar Anda.",
+    "copyright": "Hak Cipta 2002-{year} Rennie Glen Software LLC"
+  },
+  "choose": {
+    "whatNext": "Apa selanjutnya?"
+  },
+  "tools": {
+    "booklet": "Buklet",
+    "grid": "<i>Grid</i>",
```

Comment:
If Grid, Monkey, Plug-in, etc. were not italic, how confusing would it be to Indonesian readers?

Reply:
It won't be confusing at all. Readers would assume it's a name (for Grid, Monkey, and other terms).
It also applies to Plug-in, since there's no formal translation for that term.

Reply:
Yes, please remove the Html tags for this one and any string whose name doesn't include the word Html,.

However please preserve the Html tags on line 47, because its name is "introHtml".  Likewise, preserve the Html tags in all strings that are named with _Html_.

Reply:
All right, done--I've removed all HTML tags from non-Html strings.